CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.0] - 2022-06-22
```

Comment:
wtf, we are already on 0.4.0

Reply:
eh, sorry i guess iam too tired, I was looking at versions on dns-operator-aws :sob: